docs/data-sources/as_groups.md

```diff
@@ -7,7 +7,10 @@ subcategory: "Auto Scaling (AS)"
 Use this data source to get a list of AS groups.
 
 ```hcl
+variable "ep_id" {}
+
 data "flexibleengine_as_groups" "groups" {
+   enterprise_project_id = var.ep_id
```

Comment:
please delete this query parameter beacuse it should add whitelist for most users.